Directory.Packages.props

```diff
@@ -29,6 +29,8 @@
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="$(MicrosoftExtensionsDiagnosticsHealthChecksEntityFrameworkCorePackageVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="$(MicrosoftExtensionsDiagnosticsHealthChecksPackageVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Features" Version="$(MicrosoftExtensionsFeaturesPackageVersion)" />
+    <!-- Roslyn dependencies -->
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.8.0" />
```

Comment:
Correct me if I'm wrong, but I believe this means that we won't be able to run this tool if using VS older than 17.8, right? That is probably fine given targeting 8.0 already requires 17.8, but just mentioning this in case we want to add a note in the requirements somewhere.

Reply:
The tool is its own executables with its own dependencies. It doesn't matter if it is running in VS, or any other IDE. The dependencies are separate. (And yes, we only support .NET 8)

Reply:
Yup, for a sec there I thought we were running this in-proc, but then remembered we decided not to when I saw the targets file.